### DISCUSSION TITLE
A recent talk about Jupyter Book and MyST at the NumFocus summit

### DISCUSSION THREAD

**Discussion by choldgraf** [executablebooks/meta#830] [Show and tell]
I recently gave a talk about the Jupyter Book and MyST projects for the NumFocus 2022 summit. You can find a link below!

- [link to google slides](https://docs.google.com/presentation/d/1QXkfnUB-InLD2Dc5BOftJZv-psURTYGYpLn-Nl0ugTw/edit?usp=sharing)

The talk basically followed three sections:

1. Jupyter's approach to modular and composable tools that build on open standards and APIs
2. Our project's efforts at extending Jupyter use-cases for communication and publishing workflows
3. Some ideas for where these projects might evolve next, including moving to a more well-structured JS-based MyST implementation, as well as expanding this project beyond its grant-funded team and growing a broader community and governance around it.

In general people really liked the talk, and many folks were really excited about the "semantic structure" aspects of the javascript implementation. They saw a lot of potential for academic publishing, remix/re-use, etc.

Folks were concerned about whether the MyST parser / Sphinx stack would lose maintenance over time, since it is heavily used by many in the NumFocus community. This makes me think that we need to sharpen our language about next steps for our pre-existing tools to give others confidence that things like the MyST parser will continue to work and be supported as myst evolves.

I also heard from a number of groups that were building internal Jupyter Book capacity (e.g. teams that had developers in the Jupyter Book space). This included folks from NASA JPL, EconARK (cc @jstac), and AstroPy. Several folks were interested in how the governance and sustainability of the project was going to work moving forward, so I think there is an opportunity to build a broader coalition of stakeholders.

**Comment by jstac** [comment]
Awesome slides!

>  This makes me think that we need to sharpen our language about next steps for our pre-existing tools to give others confidence that things like the MyST parser will continue to work and be supported as myst evolves.

+1
